go.mod

```diff
@@ -244,6 +244,9 @@ require (
 	github.com/bep/godartsass/v2 v2.0.0 // indirect
 	github.com/bep/golibsass v1.1.1 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
+	github.com/chromedp/cdproto v0.0.0-20230802225258-3cf4e6d46a89 // indirect
+	github.com/chromedp/chromedp v0.9.2 // indirect
+	github.com/chromedp/sysutil v1.0.0 // indirect
```

Comment:
These new imports add 5MB to the slim binary, perhaps we need to consider breaking off `exp scaletest` from slim 🤔

Reply:
I can do this here 👍

Reply:
https://github.com/coder/coder/pull/9934